lldb/test/API/commands/expression/expr-from-non-zero-frame/main.c

```diff
@@ -0,0 +1,6 @@
+int func(void) {
+  __builtin_printf("Break here");
```

Comment:
This fails to compile on Windows:
https://lab.llvm.org/buildbot/#/builders/141/builds/7573

```
lld-link: error: undefined symbol: printf

>>> referenced by main.o:(func)

clang: error: linker command failed with exit code 1 (use -v to see invocation)
```
No idea why it would complain, perhaps on some platforms the builtin use here is compiled to puts and on windows it isn't?

Reply:
Sorry yes it is building now but failing:
```
FAIL: test (TestExprFromNonZeroFrame.ExprFromNonZeroFrame.test)

   Tests that we can use SBFrame::EvaluateExpression on a frame

----------------------------------------------------------------------

Traceback (most recent call last):

  File "C:\Users\tcwg\llvm-worker\lldb-aarch64-windows\llvm-project\lldb\test\API\commands\expression\expr-from-non-zero-frame\TestExprFromNonZeroFrame.py", line 29, in test

    self.assertTrue(result.GetError().Success())

AssertionError: False is not true
```
False is indeed not true, thank you test framework /s

Reply:
Skipped for now: https://github.com/llvm/llvm-project/commit/d4002b43f517fea0292bf71dccaa3d0f6dd798b9

It could be another case where we link with link.exe which can't handle DWARF information, and that's needed for the test. We will try to confirm or deny that.

Reply:
Thanks!